src/instance.ts

```diff
@@ -35,7 +34,7 @@ import {
 } from './common';
 import {Duplex} from 'stream';
 import {SessionPoolOptions, SessionPool} from './session-pool';
-import {Operation as GaxOperation} from 'google-gax';
+import {grpc, Operation as GaxOperation} from 'google-gax';
```

Comment:
Should import `CallOptions` as well.

Reply:
It will be unused. We may only need it for jsdoc, I'm not sure if it's a good idea to import it just for docs, WDYT?